libvast/src/format/pcap.cpp

```diff
@@ -106,6 +108,27 @@ const char* reader::name() const {
   return "pcap-reader";
 }
 
+vast::system::report reader::status() const {
+  using namespace std::string_literals;
+  auto stats = pcap_stat{};
+  if (auto res = pcap_stats(pcap_, &stats); res != 0)
+    return {};
+  uint64_t recv = stats.ps_recv - last_stats_.ps_recv;
+  uint64_t drop = stats.ps_drop - last_stats_.ps_drop;
+  uint64_t ifdrop = stats.ps_ifdrop - last_stats_.ps_ifdrop;
+  // Warn if at least 5% were dropped since the last report.
+  if ((drop + ifdrop) * 20 >= recv)
```

Comment:
Can we make this threshold configurable?

Reply:
It's trivial to implement, but do we want to add yet another option to the PCAP reader for this?

Reply:
In this case, yes, because everybody has different notions of what acceptable packet loss is.